### DISCUSSION TITLE
Translate correction angles into number of notches on levelling wheel

### DISCUSSION THREAD

**Discussion by shoorek83** [jneilliii/OctoPrint-BedLevelVisualizer#526] [Ideas]
Hi,

I love this plugin, especially correction angles. It let's me quickly land perfectly level my bed in no more then two /three attempts. My usual practice is to calculate corrections into number of notches on my levelling wheels using a formula [correction angle] / 360 x [total number of notches]. Unfortunately it requires me to manually calculate angles into notches every time. I believe it would be much more convenient if there was an option to enter a total number of notches on levelling wheel, so the correction matrix would return the number of notches I need to turn each wheel by.

**Comment by jneilliii** [comment]
This is requested in #469
